tests/test-stake-pool-sdk.ts

```diff
@@ -178,6 +180,99 @@ describe("test basic functionalities", () => {
       );
     });
 
+    it("it calcSolDeposit() and calcSolDepositsInverse() works correctly", async () => {
```

Comment:
Also could you move all these tests out of `describe("testnet executions", () => {` and into `describe("test basic functionalities", () => {` since they don't actually execute anything on testnet?

Reply:
shit i also just realized i nested `describe("testnet executions")` within `describe("test basic functionalities")` when it should be outside on the same level, oops. Sorry could we just fix that on this branch too?